docs/metodologia.md

```diff
@@ -0,0 +1,78 @@
+
+<p align = "justify"> &emsp;&emsp;Existem diferentes práticas exercidas pela XP, a ideia é conseguir reunir, junto da aplicação do SCRUM algumas dessas boas práticas dentro dos ritos e reuniões, que são elas o  <b>Pareamento</b>, onde dois desenvolvedores trabalham em uma mesma máquina, sendo um escrevendo o código e o outro observando para evitar e remover qualquer tipo de erro, os <b>Testes</b> onde todo código escrito deve passar por uma bateria de testes para ser validado ou não de acordo com os requisitos para entrar no código principal do projeto e a <b>Refatoração</b> com o objetivo de sempre deixar o código mais simples, mantendo sempre as funcionalidades já desenvolvidas </p>
+
+### 2.4 Lean
+
+<p align = "justify"> &emsp;&emsp; Lean é uma filosofia de gestão que data da década de 80 a partir das práticas e resultados do sistema Toyota, seu grande objetivo é reduzir o desperdício de recursos, que podem ser tempo, mão de obra, entre outros. Ou seja, busca manter a qualidade reduzindo os desperdícios, mesmo surgindo da área de manufatura, atualmente vem sendo empregado em diversos setores do mercado, como o de TI, gestão e até mesmo marketing. Muitas vezes é empregada ligada às metodologias ágeis, e é isso o que o nosso time vai buscar, o uso de apenas recursos necessários para agilizar e dar mais leveza ao processo de desenvolvimento, enquanto se aumenta o valor agregado para o cliente.</p>
+
+Alguns dos príncipios utilizados serão:
+
+* **Valor agregado**: Identificar o que realmente gera valor para o cliente, podendo assim eliminar atividades desnecessárias, preservando aquelas que agregam valor para o cliente.
+* **Redução de desperdício**: Esse príncipio será utilizado de maneira a reduzir possível mão de obra alocada inadequadamente, com a constante avaliação do processo, é possível observar a necessidade de fortalecer a equipe ou reduzir desperdício através do gerenciamento adequado dos membros.
+## 3. Ferramentas utilizadas
+
+||Ferramenta|Finalidade|
+|:-:|:-:|-|
+| <img src="./assets/metodologia/github.png" width="50px" /> | Github | Versionamento do projeto. |
+| <img src="./assets/metodologia/telegram.png" width="50px" /> | Telegram | Comunicação por texto entre os integrantes. |
+| <img src="./assets/metodologia/teams.png" width="50px" /> | Teams | Gravação das apresentações. |
+| <img src="./assets/metodologia/discord.png" width="50px" /> | Discord | Comunicação por voz entre os integrantes. |
+| <img src="./assets/metodologia/zenhub.png" width="50px" /> | Zenhub | Utilizado como quadro da metodologia kanban. |
```

Comment:
As imagens para mim estão quebradas, talvez usar o markdown funcione.

Reply:
Não será possível adicionar o markdown porque não tenho controle do tamanho das imagens, a não ser que reduza elas externamente, por isso optei por essa maneira, aparece quebrada apenas no ambiente local, quando subir pro site ela estará funcional.
Mais detalhes da discussão: [https://stackoverflow.com/questions/14675913/changing-image-size-in-markdown](https://stackoverflow.com/questions/14675913/changing-image-size-in-markdown)